parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -590,6 +610,40 @@ impl pallet_uniques::Config for Runtime {
 	type Locker = ();
 }
 
+parameter_types! {
+	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
+	pub const NftsMaxDeadlineDuration: BlockNumber = 12 * 30 * DAYS;
+}
+
+impl pallet_nfts::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type CollectionId = u32;
+	type ItemId = u32;
+	type Currency = Balances;
+	type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+	type ForceOrigin = AssetsForceOrigin;
+	type Locker = ();
+	type CollectionDeposit = CollectionDeposit;
+	type ItemDeposit = ItemDeposit;
+	type MetadataDepositBase = UniquesMetadataDepositBase;
```

Comment:
@jsidorenko 
I would maybe separete all nft deposits, and made them clear:

`UniquesMetadataDepositBase` -> `NftsMetadataDepositBase`

+ something like:
https://github.com/paritytech/cumulus/pull/2167/files#diff-67f8d30fd426048ab88d224694ba10c5f225eb66b5afdcdcb10ead450e6d2c02R267-R276

Reply:
Done